### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by boginw**
Large PR, minor changes.

### CODE REVIEW

**Comment by TobiasPalludan** [comment]
@aau-giraf/sw609f19 would you review this with me?
Here's the [review checklist](https://github.com/aau-giraf/wiki/blob/master/process_manual/code_workflow.md#code-review)

File: lib/blocs/auth_bloc.dart [hunk 1]
```diff
@@ -2,24 +2,29 @@ import 'package:rxdart/rxdart.dart';
...
 import 'package:weekplanner/providers/api/api.dart';
 
+/// All about Authentication. Login, logout, etc.
 class AuthBloc extends BlocBase {
-  Api api;
+  /// Default Constructor
+  AuthBloc(this._api);
 
-  AuthBloc(this.api);
+  Api _api;
```
Comment (msimon16) [inline lib/blocs/auth_bloc.dart]: burde api ikke være final?

**Review by MStenkaer — COMMENTED** [review]
"Document all public members" still occurs a lot of times in the project.

**Review by msimon16 — DISMISSED** [review]

**Comment by boginw** [comment]
> "Document all public members" still occurs a lot of times in the project.

This is only a warning for now

File: lib/blocs/auth_bloc.dart (hunk 1 above)
Comment (boginw) [reply lib/blocs/auth_bloc.dart]: This has been resolved

**Review by MStenkaer — DISMISSED** [review]

**Review by TobiasPalludan — APPROVED** [review]

**Review by MStenkaer — APPROVED** [review]

**Pull request merged**